src/tss2-esys/esys_crypto_mbed.c

```diff
@@ -1,4 +1,8 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
+/*******************************************************************************
+ * Copyright: 2020, Andreas Dröscher
+ * All rights reserved.
+ ******************************************************************************/
```

Comment:
What about esys_crypto_mbed.h?

Reply:
Sorry I missed that one (I'm not used to having copyright headers in each file). It's fixed with 07c34f19a7e29f6be64e2be7a0bc2747b984911d